flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/Task.java

```diff
@@ -190,7 +190,7 @@
 	/** Serialized version of the job specific execution configuration (see {@link ExecutionConfig}). */
 	private final SerializedValue<ExecutionConfig> serializedExecutionConfig;
 
-	private final ResultPartitionWriter[] producedPartitions;
+	private final ResultPartitionWriter[] partitionWithNotifiers;
```

Comment:
I think the field name could rather be `resultPartitionWriters` or `consumableNotifyingPartitionWriters`.

Reply:
I would take `consumableNotifyingPartitionWriters`